network/src/tests/peer_store.rs

```diff
@@ -53,6 +67,25 @@ fn test_ban_peer() {
     peer_store.add_connected_peer(addr.clone(), SessionType::Inbound);
     peer_store.ban_addr(&addr, 10_000, "no reason".into());
     assert!(peer_store.is_addr_banned(&addr));
+    peer_store
+        .mut_ban_list()
+        .unban_network(&multiaddr_to_ip_network(&addr).unwrap());
+    assert!(!peer_store.is_addr_banned(&addr));
+
+    for i in 0..=CLEAR_EXPIRES_PERIOD {
+        let addr: Multiaddr = format!(
+            "/ip4/{}.{}.0.{}/tcp/{}/p2p/{}",
+            rand::random::<u8>(),
+            rand::random::<u8>(),
+            rand::random::<u8>(),
+            i,
+            PeerId::random().to_base58()
+        )
+        .parse()
+        .unwrap();
+        peer_store.ban_addr(&addr, 10_000, "no reason".into());
+    }
+    assert!(peer_store.ban_list().count() >= 1024)
```

Comment:
please add a comment to explain this assert: why hardcode 1024 here/

Reply:
it should be `= 1024`, not `>= 1024`, do I understand correctly?

Reply:
yes